pkmntypes.py

```diff
@@ -22,3 +22,20 @@ def __init__(self, **kwargs):
 
 class Pokemon():
 	pass
+
+class BattleContext():
+	"""This is a class that describes the point of view of a given Pokemon on the battlefield. It provides
+	enough information for a user to make an informed decision about what turn to make next.   
+	"""
+	# will add make the docstrings comply with the sphinx format once this class is looking good
+
+	def __init__(self):
+		with open('data.json') as d:
+			data = json.load(d)
```

Comment:
```suggestion
	def __init__(self, **kwargs):
```
and change `data` to `kwargs` below